spring-cloud-stream-binder-kinesis/src/main/java/org/springframework/cloud/stream/binder/kinesis/properties/KinesisBinderConfigurationProperties.java

```diff
@@ -121,6 +131,8 @@ public void setAutoCreateStream(boolean autoCreateStream) {
 
 		private Integer timeToLive;
 
+		private Long interval = null;
```

Comment:
The same JavaDocs description, please, here.
I don't think we need this `= null`